draft/spec/index.html

```diff
@@ -913,6 +913,17 @@ <h2>Version Inventory and Inventory Digest</h2>
             Non-normative note: Storing an inventory for every version provides redundancy for this critical
             information in a way that is compatible with storage strategies that have immutable version directories.
         </blockquote>
+        <section id="conformance-of-prior-versions">
+          <h2>Conformance of prior versions</h2>
+          <p>
+              Version directories in OCFL are intended to be immutable in that existing version directories do not
+              change when new version directory is added. Each version directory within an OCFL Object
+              <span id="E103">MUST</span> conform to either the same or a later OCFL specification version as the
```

Comment:
Is there an `E103`?

Reply:
It is new. That reminds me we need to talk about versioning the validation codes file: https://github.com/OCFL/spec/issues/553

Reply:
I am not seeing a pull-request nor a direct change to [validation-codes.md](https://github.com/OCFL/spec/blob/main/validation/validation-codes.md) with the addition of `E103`. When you say, "It is new", are you saying, "We need to create `E103` in 'validation-codes.md'"? ...or should I be looking somewhere else?

Reply:
Yes, we need to add it when #553 is resolved

Reply:
Addressed with f0161002cae2c397c255e7ca19d22633018f37f6